go/carmen/carmen.go

```diff
@@ -198,6 +200,11 @@ type HistoricBlockContext interface {
 	// otherwise, the proof is returned.
 	GetProof(address Address, keys ...Key) (WitnessProof, error)
 
+	// CreateLiveDBGenesis writes a LiveDB genesis file for given block into out.
+	// The genesis file is exported from the archive for the
+	// given block and is used to sync LiveDB to that block.
+	CreateLiveDBGenesis(out io.Writer) (Hash, error)
```

Comment:
This function will eventually also need a `context.Context` to support canceling the export, right?